cmd/check.go

```diff
@@ -13,3 +18,25 @@ var checkCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(checkCmd)
 }
+
+func checkJunitAndWrite(cfg runtime.Config, results runtime.Results) error {
```

Comment:
NIT: Can we rename this to something like `writeJunitIfIEnabled` so we know the exact case causing an action?

Reply:
Also, do we need `runtime.Config` to be passed in here?

Reply:
I will check into that.